src/backend/distributed/planner/multi_explain.c

```diff
@@ -201,6 +195,15 @@ ExplainSubPlans(List *subPlanList, ExplainState *es)
 		char *queryString = NULL;
 		instr_time planduration;
 
+		if (es->format == EXPLAIN_FORMAT_TEXT)
```

Comment:
This is something that I don't know. How do you understand that we're on `VERBOSE` mode here?

Reply:
The verbose flag in the explain state.
https://github.com/postgres/postgres/blob/REL_10_STABLE/src/include/commands/explain.h#L32

Note that the Citus-specific parts are the same for verbose and normal. I just added a VERBOSE test because it nice shows the parameters of the `read_intermediate_result` call, but that's postgres' output.

Reply:
Okay, now all makes sense.